### ISSUE TITLE
an exception that I don't know how to trigger!

### ISSUE THREAD

**Issue by xywf221** [cseelhoff/RimThreaded#207]
```c#
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
  at System.Collections.Generic.Dictionary`2[TKey,TValue].get_Item (TKey key) [0x0001e] in <567df3e0919241ba98db88bec4c6696f>:0 
  at Verse.Dijkstra`1[T].Run (System.Collections.Generic.IEnumerable`1[T] startingNodes, System.Func`2[T,TResult] neighborsGetter, System.Func`3[T1,T2,TResult] distanceGetter, System.Collections.Generic.List`1[T] outDistances, System.Collections.Generic.Dictionary`2[TKey,TValue] outParents) [0x000f4] in <6326ac03f61f45efb3178f3bfd084139>:0 
  at RimWorld.InfestationCellFinder.CalculateDistanceToColonyBuildingGrid (Verse.Map map) [0x00099] in <6326ac03f61f45efb3178f3bfd084139>:0 
  at RimWorld.InfestationCellFinder.CalculateLocationCandidates (Verse.Map map) [0x00016] in <6326ac03f61f45efb3178f3bfd084139>:0 
  at RimWorld.InfestationCellFinder.TryFindCell (Verse.IntVec3& cell, Verse.Map map) [0x0000d] in <6326ac03f61f45efb3178f3bfd084139>:0 
  at RimWorld.IncidentWorker_Infestation.CanFireNowSub (RimWorld.IncidentParms parms) [0x0001f] in <6326ac03f61f45efb3178f3bfd084139>:0 
  at RimWorld.IncidentWorker.CanFireNow (RimWorld.IncidentParms parms, System.Boolean forced) [0x00178] in <6326ac03f61f45efb3178f3bfd084139>:0 
  at RimWorld.StorytellerComp+<>c__DisplayClass6_0.<UsableIncidentsInCategory>b__0 (RimWorld.IncidentDef x) [0x00020] in <6326ac03f61f45efb3178f3bfd084139>:0 
  at System.Linq.Enumerable+WhereListIterator`1[TSource].MoveNext () [0x00037] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at Verse.GenCollection.TryRandomElementByWeight[T] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] weightSelector, T& result) [0x0021e] in <6326ac03f61f45efb3178f3bfd084139>:0 
  at RimWorld.StorytellerComp_RandomMain+<MakeIntervalIncidents>d__2.MoveNext () [0x00085] in <6326ac03f61f45efb3178f3bfd084139>:0 
  at RimWorld.Storyteller+<MakeIncidentsForInterval>d__21.MoveNext () [0x00171] in <6326ac03f61f45efb3178f3bfd084139>:0 
  at RimWorld.Storyteller+<MakeIncidentsForInterval>d__20.MoveNext () [0x000a1] in <6326ac03f61f45efb3178f3bfd084139>:0 
  at RimWorld.Storyteller.StorytellerTick () [0x00042] in <6326ac03f61f45efb3178f3bfd084139>:0 
  at RimThreaded.RimThreaded.CompletePostWorkLists () [0x001ee] in <a596b092e9db4f258f9f2117d1aca92b>:0 
Verse.Log:Error(String, Boolean)
RimThreaded.RimThreaded:CompletePostWorkLists()
RimThreaded.RimThreaded:ProcessTicks()
RimThreaded.<>c:<CreateWorkerThread>b__99_0()
System.Threading.ThreadHelper:ThreadStart_Context(Object)
System.Threading.ExecutionContext:RunInternal(ExecutionContext, ContextCallback, Object, Boolean)
System.Threading.ExecutionContext:Run(ExecutionContext, ContextCallback, Object, Boolean)
System.Threading.ExecutionContext:Run(ExecutionContext, ContextCallback, Object)
System.Threading.ThreadHelper:ThreadStart()

```

**Comment by xywf221** [comment]
It often appears

**Comment by cseelhoff** [comment]
thanks for bug report. issue has been fixed and will be released with version 1.0.75
